tests/bearlib/aspects/ClassTest.py

```diff
@@ -1,5 +1,3 @@
-from coalib.bearlib.aspects import aspectclass
```

Comment:
As these were tests and did not break anything, I removed the unused imports. Will change that if it is necessary.